packages/pwa/webpack.config.js

```diff
@@ -70,66 +73,25 @@ module.exports = {
         new CleanWebpackPlugin(),
         {
             apply(compiler) {
-                compiler.hooks.compilation.tap("Store Built Files for CSP", (compilation) => {
+                compiler.hooks.compilation.tap("Update CSP - dev", (compilation) => {
```

Comment:
This only runs for dev now (I didn't want to remove CSP entirely, so I just limited to the localhost for somethings, and kept the other limitations as well).